modules/promo-code/promo-code.service.js

```diff
@@ -1,28 +1,31 @@
 const RuleError = require('../../errors/rule.error');
 const PromoCode = require('./promo-code.model');
-
 const {
   STATUS_CODES: { NOT_FOUND },
 } = require('../../consts/status-codes');
 const {
   PROMOCODE_NOT_FOUND,
-} = require('../../error-messages/promocode.messages');
+} = require('../../error-messages/promocode-messages');
 
 class PromoCodeService {
   async getAllPromoCodes() {
     const items = await PromoCode.find().exec();
+    const count = PromoCode.find().countDocuments();
 
```

Comment:
do we need an additional request to DB if we have already got the object?

Reply:
Fixed this